features/barclamp_ceph.feature

```diff
@@ -0,0 +1,33 @@
+@ceph
+Feature: Tests Ceph barclamp deployment
+  As an actor
+  I want to perform various validations
+  In order to verify the feature functionality
+
+  Background:
+    Given the chef role "ceph-mon" exists on admin node
+    And the chef role "ceph-mds" exists on admin node
+    And the chef role "ceph-osd" exists on admin node
+    And the chef role "ceph-radosgw" exists on admin node
+    And the chef role "ceph-calamari" exists on admin node
+    And the "ceph" cookbook exists on the admin node
+
+  Scenario: Ceph deployment and functioning
+    Given the barclamp proposal for "ceph" is deployed
```

Comment:
By putting the "ceph" into quotes you seem to make this step reusable for other barclamps which is not what we want as the implementation is ceph specific. By removing the quotes this should be fixed.